StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -326,7 +326,25 @@ extension CustomerSheet {
         switch customerSheetDataSource.dataSource {
         case .customerSession(let customerSessionAdapter):
             let (elementsSession, customerSessionClientSecret) = try await customerSessionAdapter.elementsSessionWithCustomerSessionClientSecret()
-            let selectedPaymentOption = CustomerPaymentOption.defaultPaymentMethod(for: customerSessionClientSecret.customerId)
+
+            var selectedPaymentOption: CustomerPaymentOption?
+
+            // read from back end
```

Comment:
I think this comment could be better!

Reply:
Changed it to "get default payment method from elements session"— is that better?